CONTRIBUTING.md

```diff
@@ -33,99 +30,111 @@ If you use [nvm](https://github.com/creationix/nvm) or [nave](https://github.com
 
-### Run the development environment
+```sh
+npm start
+```
 
-We use [Storybook](https://storybook.js.org/) for our development environment. Run `npm start` to start the Storybook server, then go to [http://localhost:9001](http://localhost:9001) in a web browser to view it.
+## Write your code
 
-## Adding icons
+Before you start writing code, we recommend familiarising yourself with the engineering convetions and squad decisions which are kept in the [decisions folder](/decisions).
 
-If you want to add icons, please discuss them with us first.
+### React components
 
-Once they're signed off, you can [raise a request](https://bit.ly/backpack-request) and attach the SVG files. If you're feeling heroic and want to make the PR yourself, just copy the correctly named SVG files to the `lg` and `sm` directories in [`@skyscanner/bpk-svgs/src/icons/`](https://github.com/Skyscanner/backpack-foundations/tree/main/packages/bpk-svgs/src/icons/) and then run `npm run build`.
+> **All Backpack components are written in Typescript or are being gradually migrated to Typescript.**
+>
+> As we're in the process of migrating all Backpack components to Typescript, we kindly request that engineers contributing changes to an existing component also migrate the respective component.
 
-## Adding a new component
+#### React version
 
-If you want to add a new component, we will need the following:
+Our current supported React version is 17.0.2, please be mindful when using React features that may not yet be supported.
 
-- Design (Figma file)
-- Associated tokens
-- Sass mixin(s)
-- React component
-- Stories
-- Tests
-- Documentation (Including main `README.md`)
+#### Design
+If you'd like to contribute a change to a React component, please first reach out to Koala team on Slack to discuss and agree on the proposed change. Make sure to add to your message the design (Figma file) and include examples for each state e.g. disabled, expanded etc. if applicable.
 
-### Design
+#### Example components
+Look at existing components for code style inspiration. Here are some good examples to follow:
+- [bpk-component-chip](./packages/bpk-component-chip/index.ts)
+- [bpk-component-button](./packages/bpk-component-button/src/BpkButtonV2/BpkButton.tsx)
 
-Figma is the preferred format for non-technical folks. We’d appreciate if you could provide an exact match of your component in Figma format together with folders for each state e.g. disabled, expanded etc.
+#### CSS
 
-### Tokens
+We use [CSS Modules](https://github.com/css-modules/css-modules) along with [BEM](http://getbem.com/) to prevent collisions and accidental overwrites in CSS.
 
-Any visual CSS parameters of the component, such as *color, margins, paddings* etc. should not live as magic numbers in the component code, but as **tokens** in the [`@skyscanner/bpk-foundations-web`](https://github.com/Skyscanner/backpack-foundations/tree/main/packages/bpk-foundations-web) package.
```

Comment:
The only reason I removed it is because we don't have the tokens in this repo anymore 😄 I was also hesitant because even though they don't live in this repo anymore, it's still relevant as this is where they would be used so I'll add it back since you mentioned it too 👍